### PR TITLE
refactor: use standard function syntax for class method

### PR DESCRIPTION
**Pull request by sherwinski**
And other stylistic tweaks

### CODE REVIEW

**Review by frederickfogerty — APPROVED** [review]
LGTM! Also I realised here that you're using a template string, so you could just easily replace those `\n`s with a real line-break if you wanted to in the future.

**Comment by sherwinski** [comment]
Ha yeah that would read a bit cleaner

File: src/api-error.js [hunk 1]
```diff
@@ -15,13 +15,11 @@ class APIError extends Error {
...
-      this.response,
-      null,
-      2,
-    )}`;
-  };
+  toString() {
+    return `${this.message}
+    Response:
+    ${JSON.stringify(this.response, null, 2)}`;
+  }
```
Comment (frederickfogerty) [inline src/api-error.js:22]: FYI, this is going to format the error messages like this:

```
[Message]
    Response:
    { .. }
```

This is probably not what you want - you'll want to remove the leading spaces on the second and third lines.

Comment (sherwinski) [reply src/api-error.js:22]: addressed in #78, granted it looks ugly now imo 😂

**Pull request merged**